environment.yml

```diff
@@ -42,7 +43,7 @@ dependencies:
   - conda-forge/label/gcc7::nn
   - opencv>=3.0.0,<=4.0.0
   - openssl
-  - pcl
+  - pcl==1.8.1
```

Comment:
If the problem with PCL is that it can't be found, this is an issue with the FindPCL script having passing the version number into the include finding call. The better solution would be to make it more flexible, whether by using the PCL-Config script that PCL ships with or by simply changing our findPCL to be more flexible with versions.